angular/src/app/services/user.service.ts

```diff
@@ -15,6 +15,7 @@ export class UserService {
 
   createUser(user: IUser): Observable<HttpResponse<object>> {
     user.locale = 'en';
+    const headers = 'headers made with HttpHeaders';
```

Comment:
why? 🤔

Reply:
excuse me please, it was a mistake on my part I did not remove the constant after testing with the interceptor